bsconfig.json

```diff
@@ -9,7 +9,7 @@
       "subdirs": true
     },
     {
-      "dir": "example",
+      "dir": "examples",
```

Comment:
Removing the `example` dir completely from compilation, since it'll totally fail the compiler w/ the new bindings.